src/pages/search/index.tsx

```diff
@@ -465,6 +472,50 @@ const Search: InferGetServerSidePropsType<typeof getServerSideProps> = ({
                       )}
                     </SlideOut>
                   )}
+                  <SlideOut showCloseButton={false}>
+                    <div className="text-sm text-text-secondary flex flex-col gap-5">
```

Comment:
We also need to add this to the `SlideOut` above (see line `468`) with a check on the `currentSlideOut` context value to ensure we are displaying the correct slide out content.

Currently if you click on the "Topics" item, this new geography slide out appears, taking UI precedent over the others.